py3status/modules/do_not_disturb.py

```diff
@@ -1,104 +1,225 @@
 # -*- coding: utf-8 -*-
 """
-Turn on and off dbus notifications.
+Toggle Do Not Disturb notifications.
```

Comment:
I think the old description was clearer

Reply:
Not all notifications uses `dbus`.

Only `xfce4-notifyd` uses `dbus` so why did we have this in first place? `:face_with_head_bandage:

Reply:
How about "Turn on and off desktop notifications"

Reply:
they all use dbus that is how they work - that is the source of their notifications